docs/Walkthrough/Installing a Package.md

```diff
@@ -2,18 +2,24 @@
 
 As explained in ["Uploading a Package"](Uploading%20a%20Package.md), packages are managed using *registries*. There is a one local registry on your machine, and potentially many remote registries elsewhere "in the world". Use `list_packages` to see the packages available on a registry:
 
-```bash
-$ python
->>> import quilt3
 
->>> quilt3.list_packages()  # list local packages
+```python
+import quilt3
+quilt3.list_packages()  # list local packages
+```
 
+```bash
```

Comment:
these outputs are no longer correct, you can one-time regenerate by actually running the code; and you will need to `list()` or otherwise unpack the generator